packages/plugin-ext/package.json

```diff
@@ -11,6 +11,7 @@
     "@theia/editor": "^1.0.0",
     "@theia/file-search": "^1.0.0",
     "@theia/filesystem": "^1.0.0",
+    "@theia/json": "^1.0.0",
```

Comment:
we don't this dependency, we only need to register configuration in core json schema storage

Reply:
@benoitf @tolusha It goes against https://github.com/eclipse-theia/theia/pull/6647, are you fine with it?

Reply:
@azatsarynnyy Do you know whether introducing such dependency will cause issues in Che? i.e. do you use json vscode built-in extensions already?